examples/research_projects/xls_r/eval.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
```

Comment:
@anton-l @Narsil - this should be the official `eval.py` script that people should take as a format for the event. Would be great if you could review it :-)